packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -128,28 +110,19 @@ Function AgentStart() {
         [string]$service_display_name
     )
 
-    if (${service_name} -eq $CWOCServiceName -And !(Test-Path -LiteralPath "${YAML}")) {
-        Write-Output "cwagent-otel-collector will not be started as it has not been configured yet."
-        return
-    }
-
     if (${service_name} -eq $CWAServiceName -And !(Test-Path -LiteralPath "${TOML}")) {
-        # If cwagent-otel-collector config exists, amazon-cloudwatch-agent default config will be suppressed
         if (Test-Path -LiteralPath "${YAML}") {
             Write-Output "amazon-cloudwatch-agent will not be started as it has not been configured yet."

```

Comment:
This if-block should probably be deleted since this refers to the ADOT yaml

Reply:
My bad for missing this one, I will remove it